app/Services/Account/Subscription/ActivateLicenceKey.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Services\Account\Subscription;
+
+use Exception;
+use Illuminate\Support\Str;
+use App\Services\BaseService;
+use App\Models\Account\Account;
+use Illuminate\Support\Facades\App;
+use Illuminate\Http\Client\Response;
+use Illuminate\Support\Facades\Http;
+use App\Exceptions\NoCustomerPortalSetException;
+use App\Exceptions\NoLicenceKeyEncryptionSetException;
+
+class ActivateLicenceKey extends BaseService
+{
+    private Account $account;
```

Comment:
It should use QueuableService

```suggestion
use App\Models\Account\Account;
use App\Services\QueuableService;
use Illuminate\Support\Facades\App;
use Illuminate\Http\Client\Response;
use Illuminate\Support\Facades\Http;
use App\Services\DispatchableService;
use App\Exceptions\NoCustomerPortalSetException;
use App\Exceptions\NoLicenceKeyEncryptionSetException;

class ActivateLicenceKey extends BaseService implements QueuableService
{
    use DispatchableService;

    private Account $account;
```